### PR TITLE
Setup gitlab PR build

### PR DESCRIPTION
**Pull request by tadaskay**
Without release/publish functionality, but should be sufficient for PR builds.

Ref #1061

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  tags:
+    - go
+  script:
+    - go get -u github.com/debber/debber-v0.3/cmd/debber
+    - bin/build_xgo linux/arm
+    - BINARY=build/myst/myst_linux_arm bin/package_debian $BUILD_VERSION armhf
+  only:
+    - master
+
+package:linux-debian-arm-raspberry:
```
Comment (soffokl) [inline .gitlab-ci.yml]:
```suggestion
package:linux-raspberry-image:
```

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+package:swagger-redoc-docker:
+  stage: build
+  tags:
+    - go
+  script:
+    - go get -u github.com/go-swagger/go-swagger/cmd/swagger/
+    - bin/swagger_generate
+    - bin/package_docker_docs
+    - docker save tequilapi:${BUILD_VERSION} | gzip > tequilapi_redoc.tgz
```
Comment (Waldz) [inline .gitlab-ci.yml]: Do we need to export docker images if all tasks are same machine now?

Comment (tadaskay) [reply .gitlab-ci.yml]: we have 4 concurrent builders for now (including 1 machine from Antanas)

Comment (tadaskay) [reply .gitlab-ci.yml]: In the next PR i'll change it to artifacts, so that they are properly published. This is interim setup (PR only)

**Pull request merged**